authentik/providers/oauth2/id_token.py

```diff
@@ -96,6 +97,8 @@ def new(
             id_token.sub = token.user.uid
         elif provider.sub_mode == SubModes.USER_ID:
             id_token.sub = str(token.user.pk)
+        elif provider.sub_mode == SubModes.USER_UUID:
+            id_token.sub = str(token.user.uuid)
```

Comment:
Fyi, by default casting a uuid to a string includes the dashes compared to using .hex, which we might want to use as this would be used for cases where a shorter sub is required

Reply:
Ah, makes sense - works well enough for my case either way, but .hex would probably have been better :)